bokeh/enums.py

```diff
@@ -42,3 +42,4 @@ def enumeration(*values):
 NamedColor = enumeration(*colors.__colors__)
 NamedIcon = enumeration(*icons.__icons__)
 MapType = enumeration("satellite", "roadmap", "terrain", "hybrid")
+DateFormat = enumeration("ATOM", "W3C", "RFC-3339", "ISO-8601", "COOKIE", "RFC-822", "RFC-850", "RFC-1036", "RFC-1123", "RFC-2822", "RSS", "TICKS", "TIMESTAMP")
```

Comment:
PEP8. I personally prefer something that is easily readable on the monitor. (even if many other parts of the file are not compliant..)